src/phoenix/server/api/auth.py

```diff
@@ -21,40 +20,13 @@ def has_permission(self, source: Any, info: Info, **kwargs: Any) -> bool:
         return not info.context.read_only
 
 
-class IsAuthenticated(Authorization):
-    message = "User is not authenticated"
-
-    def has_permission(self, source: Any, info: Info, **kwargs: Any) -> bool:
-        if info.context.token_store is None:
-            return True
-        try:
-            user = info.context.request.user
-        except AttributeError:
-            return False
-        return isinstance(user, PhoenixUser) and user.is_authenticated
+MSG_ADMIN_ONLY = "Only admin can perform this action"
 
 
 class IsAdmin(Authorization):
-    message = "Only admin can perform this action"
+    message = MSG_ADMIN_ONLY
 
     def has_permission(self, source: Any, info: Info, **kwargs: Any) -> bool:
-        if info.context.token_store is None:
-            return False
-        try:
-            user = info.context.request.user
-        except AttributeError:
-            return False
-        return (
-            isinstance(user, PhoenixUser)
-            and user.is_authenticated
-            and user.claims is not None
-            and user.claims.attributes is not None
-            and user.claims.attributes.user_role == enums.UserRole.ADMIN
-        )
-
-
-class HasSecret(BasePermission):
-    message = "Application secret is not set"
-
-    def has_permission(self, source: Any, info: Info, **kwargs: Any) -> bool:
-        return info.context.secret is not None
+        if not info.context.auth_enabled:
+            return True
```

Comment:
Don't we want to return false here? If auth is not enabled, we don't want people to be able to create system api keys for example.